packages/terra-application-navigation/src/tabs/_TabRollup.jsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import IconCaretDown from 'terra-icon/lib/icon/IconCaretDown';
+
+import TabCount from './_TabCount';
+import { enableFocusStyles, disableFocusStyles, generateKeyDownSelection } from '../utils/helpers';
+
+import styles from './Tab.module.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * Whether or not the menu display should be animated with selection.
+   */
+  isSelected: PropTypes.bool,
+  /**
+   * The selection callback for the tab.
+   */
+  onTabSelect: PropTypes.func,
+  /**
+   * The display text for the tab.
+   */
+  text: PropTypes.string,
+  /**
+   * Ref object that will be updated with a reference to the TabRollup's root element.
+   */
+  tabRef: PropTypes.shape({ current: PropTypes.instanceOf(Element) }),
+  /**
+   * Should the count notificaiton be displayed on the menu display.
+   */
+  hasChildNotifications: PropTypes.bool,
+  /**
+   * Whether or not the notification should pulse animate.
+   */
+  isPulsed: PropTypes.bool,
+  /**
+   * Boolean indicating whether or not the Tab should account for count spacing.
+   */
+  hasCount: PropTypes.bool,
+};
+
+const defaultProps = {
+  hasCount: false,
+  isPulsed: false,
+  isSelected: false,
+  hasChildNotifications: false,
+};
+
+const TabRollup = ({
+  hasCount,
+  isSelected,
+  onTabSelect,
+  tabRef,
+  hasChildNotifications,
+  text,
+  isPulsed,
+}) => (
+  <div
```

Comment:
Something seems odd here, we have a div with type="button" and role="link". Need to decide if we want to treat this as button or a link. cc/ @scottwilmarth

Reply:
Talked with @dkasper-was-taken. They will be meeting with @scottwilmarth to discuss accessibility of the component today.